NuGet-Packages.md

```diff
@@ -0,0 +1,51 @@
+NuGet packages are a convenient way to provide developers with official builds of our project while enabling
+
+See [our plans for NuGet (#85)](https://github.com/Microsoft/ChakraCore/issues/85).
+
+# NuGet Feed (Stable Builds)
+
+_Coming soon! (Check back when we finalize the `release/1.2` milestone.)_
+
+# MyGet Feed (Preview Builds)
+
+We publish preview builds of our NuGet package at the
+[Microsoft.ChakraCore](https://www.myget.org/feed/chakracore-preview/package/nuget/Microsoft.ChakraCore)
+MyGet Feed.
+
+Currently, packages on the MyGet feed are built from the `release/1.2` branch.
+We have plans to expand this to preview packages built from any branches under active development
+(such as `master`). Builds are published regularly at a rate consistent with changes to the branch
+(daily in active branches, perhaps less often in stabilization branches).
+
+# Grabbing Preview Binaries from NuGet Packages
+
+NuGet packages are .zip files with a different extension. To extract files from a NuGet package,
+simply rename the file.
+
+`rename ChakraCore.nuget ChakraCore.zip`
+
+Then you can open the file in your favorite zip file browser and extract the appropriate binaries
+for your system architecture (currently supported are x64 and x86).
+
+```
```

Comment:
Probably a good idea to do so. I'm going to leave this until we try out the packages and figure out the best way to use the NuGet package as a dependency for C++ projects. We can probably fix this problem all at once.

In order to use the ChakraCore.dll with the included ch.exe, the .lib files are not needed. Additionally, TypeScript team has embedded ChakraCore.dll without the lib files AFAIK. They may have downloaded the headers to develop against but I don't think they needed the .lib file.

/cc TypeScript folks - @DanielRosenwasser @mhegazy @paulvanbrenk can any of you comment on how difficult it was to develop against and use ChakraCore.dll from our MyGet packages as a component of tc.exe?